keylime/policy/sign_runtime_policy.py

```diff
@@ -0,0 +1,129 @@
+"""Module to assist with signing Keylime runtime policies using DSSE."""
+
+import argparse
+import logging
+from typing import TYPE_CHECKING, Any, Optional, Union
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
+
+from keylime.dsse import dsse, ecdsa, x509
+
+if TYPE_CHECKING:
+    # FIXME: how to make mypy and pylint happy here?
+    _SubparserType = argparse._SubParsersAction[argparse.ArgumentParser]  # pylint: disable=protected-access
+else:
+    _SubparserType = Any
+
+logger = logging.getLogger("policy.sign_runtime_policy")
+
+KEYLIME_PAYLOAD_TYPE = "application/vnd.keylime+json"
+KEYLIME_DEFAULT_EC_KEY_FILE = "keylime-ecdsa-key.pem"
+
+
+def get_arg_parser(create_parser: _SubparserType, parent_parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
+    """Perform the setup of the command-line arguments for this module."""
+    sign_p = create_parser.add_parser("runtime", help="create runtime policies", parents=[parent_parser])
+
+    sign_p.add_argument(
+        "-o",
+        "--output",
+        dest="output_file",
+        required=False,
+        help="The output file path for the DSSE-signed policy",
+        default="/dev/stdout",
+    )
+    sign_p.add_argument(
+        "-r",
+        "--runtime-policy",
+        dest="sign_policy",
+        required=True,
+        help="The location of the runtime policy file",
+        default="",
+    )
+    sign_p.add_argument(
+        "-k",
+        "--keyfile",
+        dest="keyfile",
+        required=False,
+        help="The private key to sign the policy with",
+        default="",
+    )
+    sign_p.add_argument(
+        "-p",
+        "--keypath",
+        dest="keypath",
+        required=False,
+        help="The filename to write the created private key, " "if one is not provided via the --keyfile argument",
```

Comment:
I think `black` does not like strings split in multiple lines, but when it puts them together is creates weird quoting.

Reply:
TBH, I just follow the linter rules, which means prefer long strings over splitting into multiple lines.

Reply:
Fixed.